src/ui/main_ui.py

```diff
@@ -117,9 +117,9 @@ def setupUi(self, MainWindow):
         self.ConTitle.setFont(font)
         self.ConTitle.setObjectName("ConTitle")
         self.verticalLayout_4.addWidget(self.ConTitle)
-        self.graphicsView = QtWidgets.QGraphicsView(self.ConFrame)
-        self.graphicsView.setObjectName("graphicsView")
-        self.verticalLayout_4.addWidget(self.graphicsView)
+        self.output1 = QtWidgets.QTextEdit(self.ConFrame)
```

Comment:
This seems to be a widget change in the ui file, but I don't see main.ui added to this commit. Please make sure to not manually change this file at all. 

Also, the UI code I sent to you in slack contains an example of a very robust console @avdarekar. If you want any reference for this, you can use that directly.

Reply:
Sorry about that, the Console was a graphics widget for some reason, and I wasn't sure how to change the widget to a textedit widget without changing the .py file itself.